packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -2273,6 +2280,7 @@ export class LitNodeClientNodeJs extends LitCore {
         params.resourceAbilityRequests.map((r) => r.resource)
       );
     let expiration = params.expiration || LitNodeClientNodeJs.getExpiration();
+    let nonce = this.latest_blockhash || generateNonce();
```

Comment:
isnt the point to not generate a local nonce? if the plan is to remove this once the nodes have merged support for this then we should have a task to follow up and throw an error if the `latest_blockhash` is undefined. Otherwise the work on the nodes can be bypassed.

Reply:
Exactly, here's the ticket: https://linear.app/litprotocol/issue/LIT-1935/update-sdk-to-throw-error-if-blockhash-form-nodes-is-undefined